docker-compose.yml

```diff
@@ -28,6 +28,24 @@ services:
     ports:
       - ${PORT__HEALTHCHECK_METRICS:-7300}:7300
 
+  eigenda-proxy:
+    platform: linux/amd64
+    image: us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/eigenda-proxy:v1.4.1
+    restart: unless-stopped
+    stop_grace_period: 5m
+    entrypoint: /scripts/start-eigenda-proxy.sh
+    volumes:
+      - eigenda-data:/data
+      - ./scripts/:/scripts
+    ports:
+      - ${PORT_EIGENDA_PROXY:-4242}:4242
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    deploy:
+      # If EIGENDA_PROXY_ENDPOINT is unset, then we set replicas to 0,
+      # otherwise use the value of EIGENDA_PROXY_ENDPOINT which should be 1.
```

Comment:
Do you mean `USE_LOCAL_EIGENDA_PROXY`? I don't think `EIGENDA_PROXY_ENDPOINT` should ever be 1 according to the comments in the env file.

Reply:
```suggestion
      # If USE_LOCAL_EIGENDA_PROXY is unset, then we set replicas to 0,
      # otherwise use the value of USE_LOCAL_EIGENDA_PROXY which should be 1.
```